docs/release-info/release-notes.md

```diff
@@ -57,50 +57,689 @@ For tips about how to write a good release note, see [Release notes](https://git
 
 This section contains important information about new and existing features.
 
+### Improved native queries
+
+Native queries can now group on nested columns and arrays.
```

Comment:
Grouping on arrays was supported before Druid 30. It should be nested arrays. 
```suggestion
Native queries can now group on supported complex columns and nested arrays.
```

Reply:
Similar functionality is also added in MSQ as well. MSQ can now group on complex columns. However, MSQ doesn't inherently support nested arrays, therefore we can't say MSQ supports grouping on nested arrays.

Reply:
Updated.